src/api/common/interceptors/metrics.interceptor.ts

```
@@ -0,0 +1,79 @@
+  CallHandler,
+  ExecutionContext,
+  Injectable,
+  NestInterceptor,
+} from '@nestjs/common';
+import { InjectMetric } from '@willsoto/nestjs-prometheus';
+import { Counter, Histogram } from 'prom-client';
+import { Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+
+function normalizePath(path: string): string {
+  return path
```

ESLint

[error] Insert `(⏎····`

(prettier/prettier)